x/mongo/driver/session/server_session.go

```diff
@@ -11,15 +11,16 @@ import (
 
 	"crypto/rand"
 
-	"go.mongodb.org/mongo-driver/x/bsonx"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/x/bsonx/bsoncore"
 	"go.mongodb.org/mongo-driver/x/mongo/driver/uuid"
 )
 
 var rander = rand.Reader
 
 // Server is an open session with the server.
 type Server struct {
-	SessionID bsonx.Doc
+	SessionID bson.Raw
```

Comment:
would it be better to just use bsoncore.Document here? 1 less import...

Reply:
Wouldn't we still have to import `bsonx/bsoncore`?

Reply:
yeah, but you wouldn't need `bson`.

Reply:
Changed.